webapp/portlet/src/main/resources/locale/portlet/dlpQuarantine/dlpQuarantine_en.properties

```diff
@@ -5,11 +5,14 @@ documents.dlp.quarantine.keywordDetected=Keyword Detected
 documents.dlp.quarantine.createdDate=Created Date
 documents.dlp.quarantine.actions=Actions
 documents.dlp.quarantine.author=Author
-documents.dlp.quarantine.previewDownload=Preview / download the document
-documents.dlp.quarantine.validateDoc=Validate the document
-documents.dlp.quarantine.deleteDoc=Delete the document
-documents.dlp.message.confirmDelete=Are you sure you want to delete this document?
-documents.dlp.title.confirmDelete=Delete document
-documents.dlp.button.ok=OK
-documents.dlp.button.cancel=Cancel
-documents.dlp.error.UnknownServerError=An unknown error happened when processing operation. Please contact your administrator.
+
+items.dlp.quarantine.previewDownload=Preview / download the document
+items.dlp.quarantine.validate=Validate the document
+items.dlp.quarantine.delete=Delete the document
+items.dlp.message.confirmRestore=Are you sure you want to restore this document?
+items.dlp.title.confirmRestore=Restore document
+items.dlp.message.confirmDelete=Are you sure you want to delete this document?
+items.dlp.title.confirmDelete=Delete document
```

Comment:
```suggestion
items.dlp.quarantine.previewDownload=Preview / download this item
items.dlp.quarantine.validate=Validate item
items.dlp.quarantine.delete=Delete item
items.dlp.message.confirmRestore=Are you sure you want to restore this item?
items.dlp.title.confirmRestore=Restore item
items.dlp.message.confirmDelete=Are you sure you want to delete this item?
items.dlp.title.confirmDelete=Delete item
```

Reply:
Ce est mis dans une autre nouvelle tache.

Reply:
Here you modify this keys. So we can modify the value in this PR